commands/next_lint_error.py

```diff
@@ -57,16 +57,25 @@ def _harvest_next(self) -> int:
             self.window.active_view().sel()[0].begin()
         )
         lines = set([])
+        next_change = None
         vid = self.window.active_view().id()
         for error_type in ['ERRORS', 'WARNINGS', 'VIOLATIONS']:
             for line, _ in ANACONDA[error_type].get(vid, {}).items():
                 lines.add(int(line))
 
-        lines = set(sorted(lines))
+        lines = list(set(sorted(lines)))
         if not len(lines):
             return None
 
-        if cur_line and list(lines)[-1] > cur_line:
-            lines_list = [l for l in lines if l > cur_line]
+        if not cur_line:
+            return None
+
+        if lines[-1] > cur_line:
+            for l in lines:
+                if l > cur_line:
+                    next_change = l
+                    break
+        else:
+            next_change = lines[0]
 
-        return lines_list[0]
+        return next_change
```

Comment:
We can achieve the same result and have a much more cleaner code just with a one line conditional return
```python
return lines_list[0] if len(lines_list) > 0 else lines.pop()
```